tests/scripts/all.sh

```diff
@@ -3896,10 +3897,11 @@ component_test_aesni () { # ~ 60s
     msg "AES tests, test AESNI only"
     make clean
     make CC=gcc CFLAGS='-Werror -Wall -Wextra -mpclmul -msse2 -maes'
-    strings ./programs/test/selftest | grep -q "AES note: using AESNI"
-    strings ./programs/test/selftest | not grep -q "AES note: built-in implementation."
     ./programs/test/selftest aes | grep -q "AES note: using AESNI"
     ./programs/test/selftest aes | not grep -q "AES note: built-in implementation."
+    strings ./programs/test/selftest | grep -q "AES note: using AESNI"
```

Comment:
`strings` works, but I don't think it's needed - why not simply do:

`grep -q "AES note: using AESNI" ./programs/test/selftest`